scheduler/docs/configuration.adoc

```diff
@@ -340,6 +342,15 @@ Example optimizer config:
 `:retry-limit`::
   This limits the number of retries a job is allowed to request. Something in the low tens is often more than sufficient.
 
+[[estimated_completion_constraint]]
+==== Estimated Completion Constraint
+`:expected-runtime-multiplier`::
+ This is a configurable constant factor which will multiply the expected runtime on each job to compute
+ when the job will complete. For instance, if `estimated-runtime-multiplier` is 2.5 and a job has an expected
+ runtime of 60000 ms, it will not be scheduled on a host which will die in 2.5 minutes.
+`:host-lifetime-mins`::
+ This is the expected lifetime of hosts in the cluster. Each host should have a `host-start-time` attribute in it's mesos offer
```

Comment:
What happens if hosts don't have the `host-start-time` in their offers?

Reply:
In that case, the job would be eligible to be scheduled on the host. I updated the doc a bit to hopefully clarify.